app/assets/stylesheets/layout/_default_list.scss

```diff
@@ -1,10 +1,10 @@
-.authors-list {
+.default-list {
```

Comment:
I would to have semantically meaningful names in the markup, and I think we don't actually need an extra `div`. How about something like:

https://github.com/taylorrf/rubybookshelf/compare/display_covers_books_on_lists_page...display_covers_books_on_lists_page_class_names